### DISCUSSION TITLE
Block a PR for a package from being mergeable (draft or commented)

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#31648] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

Is it possible using `packageRules` to set a PR as draft or perhaps put a comment on it upon creation? Something that will block the PR from being mergeable until user action is taken.

I see the `draftPR` config, but that looks like it would apply to all PRs.

**Comment by rarkins** [comment, marked as answer]
`draftPr` can be configured using `packageRules`, per-dependency if desired
